tests/core.test.ts

```diff
@@ -0,0 +1,377 @@
+import { MPPAlert, Severity, alertType } from '../src/core/alert';
```

Comment:
Rewrite the description of all the tests to more understandable versions.
Its pretty hard to understand what the tests is doing from just trying to read the description.

Reply:
done